app/views/application/_header.html.erb

```diff
@@ -12,6 +12,12 @@
               <%- end %>
             </a>
           </div>
+          <% unless current_user %>
+            <div class='auth-buttons'>
+              <a href="<%= path "/signup"%>" class='btn btn-primary btn-small signup-button'><%= I18n.t('sign_up') %></a>
```

Comment:
Shouldn't there be some logic here to ensure that users can sign up? For example the front end app does this:

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/controllers/application.js.es6#L9

Reply:
Good catch, done!